lib/stdlib/src/gen_server.erl

```diff
@@ -690,10 +691,10 @@ handle_msg({'$gen_call', From, Msg}, Parent, Name, State, Mod, HibernateAfterTim
     Result = try_handle_call(Mod, Msg, From, State),
     case Result of
 	{ok, {reply, Reply, NState}} ->
-	    reply(From, Reply),
+	    _ = reply(From, Reply),
```

Comment:
These are not needed now when reply return ok.
So remove them.

I was talking about the global.erl warnings that will emerge when you define type for multicall in your other commit.
Also not backwards compatible, so this must be on the master branch but I can fix that.

Reply:
Oh. I saw [this line](https://github.com/erlang/otp/blob/master/lib/stdlib/src/gen_server.erl#L735) and taught it's `reply/2` and changed them :-S